src/sentry/integrations/middleware/hybrid_cloud/parser.py

```diff
@@ -56,10 +55,6 @@ def __init__(
 class BaseRequestParser:
```

Comment:
i think if this inherits from ABC and we make the `provider` and `webhook` into abstract properties, i believe you can use self.provider instead of having to add the `get_provider()` function